samples/nvidia-resnet/pipeline/persistent-volume-claim.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
```

Comment:
Yeah it would be nice if we can mount persistent volumes within the pipeline but I'm not sure how to do that besides running `kubectl create -f volume.yaml` in the host system like what we did with the `mount_persistent_volume.sh` script.

Reply:
/cc @hongye-sun